### ISSUE TITLE
[BUG] enterInUrlbarToRefresh.uc.js not working on Firefox 106?

### ISSUE THREAD

**Issue by ericpa06** [aminomancer/uc.css.js#57]
Hi, I think enterInUrlbarToRefresh is not working on Firefox 106 (Firefox 106.0b2). You press enter and nothing happens, not even normal run-of-the-mill URLs without the **#** on them.  I might be something wrong with my setup, but it's worth an investigation I think. As always, thank you very much for your attention and sorry to bother.

**Comment by sakanatha** [comment]
Yes its not

**Comment by aminomancer** [comment]
Any browser console logs?

**Comment by sakanatha** [comment]
No

**Comment by Sneakpeakcss** [comment]
I also had this problem when i was testing animateContextMenus, but i thought it was only because i did a really fast installation and some scripts were interfering with each other.

Turns out the problem still happens when enterInUrlbarToRefresh.uc.js is the only script.

Error only shows when making an action in urlbar, not on launch:
![image](https://user-images.githubusercontent.com/77424331/191623522-fde6f36d-623c-44a9-aeed-bc0691dbb92a.png)

This happens whenever you press enter in urlbar, use paste and go, or even choose url with mouse in dropdown.

<sub>Firefox Nightly (107.0a1)</sub>

**Comment by aminomancer** [comment]
Yeah looks like it needs to be updated for ESMification. I've just been pretty busy and don't test those scripts frequently

**Comment by ericpa06** [comment]
Thank you [aminomancer](https://github.com/aminomancer), [sakanatha](https://github.com/sakanatha) and [Sneakpeakcss](https://github.com/Sneakpeakcss).
